content/en/blog/_posts/2023-04-14-improving-support-go-updates.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: blog
+title: "Improving Support: Go Updates"
+date: 2023-04-14
```

Comment:
How about publication on the 6th or 7th of April? 
Pick either of those (and please also change the path to match).

Reply:
updated to 2023-04-06